src/app/(protected)/account/settings/page.tsx

```diff
@@ -52,8 +85,15 @@ const Setting = () => {
                   </FormGroupField>
                 </FormGroup>
                 <FormGroup>
+                  <Description>
+                    {isFocused.displayName &&
+                      'Display name should between 4 to 15 characters, only letters, numbers, dashes and underscore are allowed'}
```

Comment:
we probably think of abstracting this magic string, and make it shareable between different pages

Reply:
add share styled component
<img width="871" alt="image" src="https://github.com/user-attachments/assets/3c3690fe-3f8a-4444-a390-381eba5ac64b">